libbpf-cargo/src/test.rs

```diff
@@ -518,7 +521,96 @@ fn test_skeleton_basic() {
     "#}
     .to_string();
 
-    let () = build_rust_project_from_bpf_c(&bpf_c, &rust);
+    build_rust_project_from_bpf_c(&bpf_c, &rust);
+}
+
+#[test]
+fn test_skeleton_generate_bpf_objs_section() {
+    let bpf_c = indoc! {r#"
+        #include "vmlinux.h"
+        #include <bpf/bpf_helpers.h>
+
+        struct {
+                __uint(type, BPF_MAP_TYPE_HASH);
+                __uint(max_entries, 1024);
+                __type(key, u32);
+                __type(value, u64);
+        } mymap SEC(".maps");
```

Comment:
Let's stick with 4 spaces for indentation per level, please.
```suggestion
        struct {
            __uint(type, BPF_MAP_TYPE_HASH);
            __uint(max_entries, 1024);
            __type(key, u32);
            __type(value, u64);
        } mymap SEC(".maps");
```